code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -33,7 +33,7 @@
 	if (autotraitor_cooldown)
 		autotraitor_cooldown--
 	else
-		autotraitor_cooldown = 900//15 minutes
+		autotraitor_cooldown = 450//15 minutes
```

Comment:
🤔

Reply:
That was supposed to be changed in #21778

Why did no one catch it then?

Reply:
Isn't that 7.5 minutes?

Update the comment.

Reply:
No, the process is only once every 2 seconds, so it's the intended length of time